scripts/us_census/acs5yr/subject_tables/common/generate_col_map.py

```diff
@@ -0,0 +1,157 @@
+        # corresponding property 'p'
+        # Eg:
+        # 'Estimate!!Total!!Total civilian noninstitutionalized
+        # population!!SEX!!Male' is the name of the column and the value you we want
+        # is the gender 'Male'. This pattern repeats for all columns
+        # To verify this assumption for specific census tables, please check the
+        # table shells at: https://www2.census.gov/programs-surveys/acs/tech_docs/table_shells/2019/
+        val_prop_col = part_list[-1]
+
+        for part in part_list:
+            # set the base for special cases like median, etc.
+            if not measurement_assigned and 'measurement' in self.features.keys(
+            ):
+                if part in self.features['measurement'].keys():
+                    stat_var.update(self.features['measurement'][part])
+                    measurement_assigned = True
+
+        # set the default statVar definition
+        if not measurement_assigned and 'measurement' in self.features.keys(
+        ) and '_DEFAULT' in self.features['measurement'].keys():
+            stat_var.update(self.features['measurement']['_DEFAULT'])
+
+        # set the populationType attribute
+        stat_var['populationType'] = self._get_population_type(part_list)
+
+        # associate pvs to stat_var
+        if val_prop_col not in self.features['measurement'].keys():
+            if 'pvs' in self.features.keys():
+                #p = property, k = dictionary of substrings in column that points to a propertyValue
+                for p, k in self.features['pvs'].items():
+                    for c, v in k.items():
+                        ## check if the current key of dict matches with any substring from the tokens of a column name
+                        for part in part_list:
+                            # check if column substring matches with a column token
+                            if c in part:
+                                stat_var[p] = v
+                                # add inferred properties from property 'p' to the stat_var
+                                if 'inferredSpec' in self.features.keys(
+                                ) and p in self.features['inferredSpec'].keys():
+                                    stat_var.update(
+                                        self.features['inferredSpec'][p])
+
+                            # check if column substring matches with any
+                            # substring within column token. Applicable when
+                            # only a substring in the column's token
+                            # Eg: Civilian noninstitutionalized population 16 to
+                            # 64 years when specified as Civilian
+                            # noninstitutionalized population, in order to make
+                            # it a generic 'c' to map all columns withi
+                            # 'Civilian' and 'noninstitutionalized'
+                            if c in part.split():
```

Comment:
if c in part: in line 99 above
should already be returning true if c is a substring of part as well.

If that is not the intention, we should check for equality instead of 'in'